courses/models.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+from uuidfield import UUIDField
+
+class Course(models.Model):
+    """
+    An edX course which is purchaseable.
```

Comment:
`purchaseable` should be `purchasable` though I couldn't tell the difference without PyCharm's hint